examples/wallet/common.go

```diff
@@ -2,16 +2,40 @@ package main
 
 import (
 	"encoding/json"
-	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/url"
+	"path"
 
 	"github.com/kaspanet/kasparov/apimodels"
 	"github.com/pkg/errors"
 )
 
+const (
+	getUTXOsEndpoint        = "utxos/address"
+	sendTransactionEndpoint = "transaction"
+)
+
+// resourceURL returns a full concatenated URL from the base
+// kasparov server URL and the given path.
+func resourceURL(kasparovAddress string, pathElements ...string) (string, error) {
+	kasparovURL, err := url.Parse(kasparovAddress)
+	if err != nil {
+		return "", errors.WithStack(err)
+	}
+	newPathElements := make([]string, len(pathElements)+1)
+	newPathElements[0] = kasparovURL.Path
+	copy(newPathElements[1:], pathElements)
+	kasparovURL.Path = path.Join(newPathElements...)
```

Comment:
Instead of creating newPathElements, copying pathElements into it, etc, why not simply 
```
kasparovURL.Path = path.Join(append(kasparovURL.Path, pathElements...))
```